scalableminds/webknossos
# Rename Resolution to Mag

In this PR, the term "resolution" is replaced with "mag"/"magnification" in a lot of places.
- For the backend, see [this comment below](https://github.com/scalableminds/webknossos/pull/8111#issuecomment-2399313689). The DB schema was adjusted, too.
- In the frontend code, method names, their parameters, types, properties, error messages and console prints were adjusted. Function or local scoped variables were not adjusted, because this would make the changes way too large to review. As I am not familiar with the shader code, I didnt change `*.glsl.ts` files either. 
- In the UI, the term "resolution" is only being used in the context of animations after this PR. Other than that, "magnification" or sometimes "mag" is (or should be) used in toasts, tooltips, modals, sliders etc.

### URL of deployed dev instance (used for testing):
- https://magalltheway.webknossos.xyz/

### Steps to test:
- Test that everythings works as before by trying to test features that depend on mags. 
- This includes:
    - Opening an annotation, zooming in and out
    - Creating an annotation with restricted mags
    - Creating a volume layer with restricted mags
    - Testing all tools in the toolbar 
    - Create & Edit task type with magRestrictions, create task + open it, should only have the allowed mags
    - reload dataset list from disk

### TODOs:
- [x] rename files, too
- [x] fix frontend tests
- [x] add some explanation to info tab tooltip

backup branch: https://github.com/scalableminds/webknossos/tree/backup-rename-resolution-to-mag

### Issues:
- contributes to #7378

------
